src/index.html

```diff
@@ -12,7 +12,13 @@
   </head>
   <body>
 
+
+
+  <biggive-hero-image slug="My Slug" main-title="This is my title" teaser="this is the intro text which I want to add"></biggive-hero-image>
```

Comment:
Suggest we start using Storybook. atm you and @alihejazi are both updating this sample file which is just going to mean merge conflict resolution work for no benefit, very shortly.